bin/submission_new.py

```diff
@@ -218,14 +218,12 @@ def extract_top_metadata(self):
 		available_columns = [col for col in columns if col in self.metadata_df.columns]
 		return self.metadata_df[available_columns].to_dict(orient='records')[0] if available_columns else {}
 	def extract_biosample_metadata(self):
-		columns = ['bs_package','isolate','isolation_source','host_disease','host','collected_by','lat_lon',
-				   'sex','age','geo_location','organism','purpose_of_sampling', 'race','ethnicity','sample_type']  # BioSample specific columns
+		columns = ['bs_package','strain','isolate','host_disease','host','collected_by','lat_lon','geo_location','organism','sample_type','collection_date']  # BioSample specific columns
 		available_columns = [col for col in columns if col in self.metadata_df.columns]
 		return self.metadata_df[available_columns].to_dict(orient='records')[0] if available_columns else {}
 	def extract_sra_metadata(self):
-		columns = ['instrument_model','library_construction_protocol','library_layout','library_name','library_selection',
-				   'library_source','library_strategy','nanopore_library_layout','nanopore_library_protocol','nanopore_library_selection',
-				   'nanopore_library_source','nanopore_library_strategy','nanopore_sequencing_instrument']  # SRA specific columns
+		columns = ['illumina_sequencing_instrument','illumina_library_protocol','illumina_library_layout','illumina_library_selection',
+				   'illumina_library_source','illumina_library_strategy']  # SRA specific columns
```

Comment:
I want to keep the nanopore ones...I'll need to handle how to handle them in XML files later

Reply:
Added them back in.